quesma/quesma/schema_transformer.go

```diff
@@ -394,6 +456,7 @@ func (s *SchemaCheckPass) Transform(queries []*model.Query) ([]*model.Query, err
 			Transformation     func(*model.Query) (*model.Query, error)
 		}{
 			{TransformationName: "PhysicalFromExpressionTransformation", Transformation: s.applyPhysicalFromExpression},
+			{TransformationName: "FullTextFieldTransformation", Transformation: s.applyFullTextField},
```

Comment:
Maybe, it would be good to add at least one unit test that would test this particular transformation?